src/BlockTypes/Cart.php

```diff
@@ -222,9 +222,23 @@ protected function deep_sort_with_accents( $array ) {
 			return $array;
 		}
 
-		if ( is_array( reset( $array ) ) ) {
-			return array_map( [ $this, 'deep_sort_with_accents' ], $array );
-		}
+		$array_without_accents = array_map(
+			'remove_accents',
+			array_map(
+				'wc_strtolower',
+				array_map(
+					'html_entity_decode',
+					array_map(
+						function ( $element ) {
+							if ( is_array( $element ) ) {
+								return $this->deep_sort_with_accents( $element );
+							}
+						},
+						$array
+					)
+				)
+			)
+		);
 
 		$array_without_accents = array_map( 'remove_accents', array_map( 'wc_strtolower', array_map( 'html_entity_decode', $array ) ) );
```

Comment:
Good catch. This should be executing on the first `$array_without_accents`.

Reply:
And actually this line could probably be removed completely because it's handled by the previous lines.